apiExamples/required.html

```diff
@@ -1,6 +1,6 @@
-<auro-radio-group required>
+<auro-radio-group required setCustomValidityValueMissing="value missing">
```

Comment:
We need to demo this behavior with and without setting `setCustomValidityValueMissing`

Reply:
We haven't done this in other components, is there a reason we should now?